README.md

```diff
@@ -22,7 +28,63 @@ Or install it yourself as:
 
 ## Usage
 
-TODO: Write usage instructions here
+The current main usage of `FastStats` is to track and build means while iterating over data sets.
+
+Example: Let's say you want to get some means for social media posts.
+
+You can track several metrics with `FastStats::Means`:
+
+```ruby
+# Say you have some PostFetcher that is an enumerator for your posts.
+means = FastStats::Means.new
+
+PostFetcher.each do |post|
+   # do stuff with post
+   means.add "likes", post["like_count"]
+   means.add "comments", post["share_count"]
+end
+
+means.summary
+# =>
+#{
+#   "foo_arithmetic"=>4.888888888888889,
+#   "foo_geometric"=>3.345423581422162,
+#   "bar_arithmetic"=>13.636363636363637,
+#   "bar_geometric"=>7.1824970648723765
+#}
+
+# You can also pass a round: value
+means.summary round: 2
+# =>
+#{
+#   "foo_arithmetic"=>4.89,
+#   "foo_geometric"=>3.35,
+#   "bar_arithmetic"=>13.64,
+#   "bar_geometric"=>7.18
+#}
+
+```
+
+You can track an individual metric with `FastStats::Mean`:
+
+```ruby
+mean = FastStats::Mean.new name: "foobar"
+10.times { |i| mean << i }  # or mean.add(i)
+
+puts mean.arithmetic # round: 2
```

Comment:
The `round 2` line comments here are unclear in intent

Reply:
Ya they kinda are aren't they.

Reply:
Yeah...the default is 15 so it's not documenting that, so I'm guessing you're just implying you could pass that option? But it's not clear to me.

Reply:
I was also confused by this comment. I'd leave it out altogether or show it actually being used
```ruby
puts mean.geometric
# => 3.597297064377001
puts mean.geometric round: 2
# => 3.6
```